calculator.c

```diff
@@ -0,0 +1,54 @@
+://Clarke Littlejohn
```

Comment:
Your code didn't compile because of this very simple and accidental syntax error! Be careful next time! You must have accidentally added it while working with vim.